.github/workflows/pylint-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Pylint
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize, edited]
+
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+    - name: Analysing the code with pylint
+      run: |
+        pylint ./dags --rcfile=scripts/.pylintrc --fail-under=9
```

Comment:
Can you run `scripts/code-style.sh` here instead and check for diffs? I'd also like to add JSonnet styling there too (#149)

Reply:
Sure thing.